--- inc/assets/js/tc-scripts.js	2014-11-24 18:29:14.758984428 +0100
+++ inc/assets/js/tc-scripts.new.js	2014-11-24 18:32:55.583977904 +0100
@@ -2514,10 +2514,12 @@
         left                = $("#main-wrapper .container " + LeftSidebarClass),
         right               = $("#main-wrapper .container " + RightSidebarClass);
 
+    var reordered = false;
+    
     function BlockPositions() {
         //15 pixels adjustement to avoid replacement before real responsive width
         WindowWidth = $(window).width();
-        if ( WindowWidth > 767 - 15 ) {
+        if ( WindowWidth > 767 - 15 && reordered ) {
             //$(window).width();
             if ( $(left).length ) {
                 $(left).detach();
@@ -2528,7 +2530,8 @@
                 $(right).detach();
                 $(wrapper).append($(right));
             }
-        } else {
+            reordered = false; //this could stay in both if blocks instead
+        } else if ( WindowWidth <= 767 - 15 && ! reordered ) {
             if ( $(left).length ) {
                  $(left).detach();
                 $(content).detach();
@@ -2538,6 +2541,7 @@
                 $(right).detach();
                 $(wrapper).append($(right));
             }
+            reordered = true; //this could stay in both if blocks instead
         }
     }//end function*/
 
@@ -2545,7 +2549,7 @@
     if ( 1 == TCParams.ReorderBlocks ) {
         //trigger the block positioning only when responsive
         WindowWidth = $(window).width();
-        if ( WindowWidth <= 767 - 15 ) {
+        if ( WindowWidth <= 767 - 15 && ! reordered ) {
             BlockPositions();
         }
 
